### DISCUSSION TITLE
장시간 진행되는 모임 시간에 대해 2시간 이내로 할 수 있는 아이디어 필요

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#356] [General]
사실 저는 2시간 이내에 끝낼 수 있는 걸 목표로 하고 있는데
서로 논의 내용이 길어지다 보면 중간에 끊을 수는 없기 때문에 3시간이나 되는 시간을 단축하기 위한 아이디어가 필요하긴 할 것 같습니다.

개인 별 발표 시간 10분 제한도 방법일 수 있는데, 너무 짧은 시간일 수도 있다는 생각도 들어서 
우선 제가 진행 역량을 강화하는 측면에서 의식적으로 빠른 진행을 해보도록 하겠습니다.
그래도 2시간을 넘는다면 그때 뭔가 규칙을 정해서 추가하는 식으로 하겠습니다.

**Comment by wooyaggo86** [comment]
개인적으로 저는 2시간 넘어도 괜찮은데.. 한국에 계시는 참가자 분들의 수면이 걱정됩니다😅

**Reply by yeslee-v** [reply]
ㅎㅎㅎ 12시가 다가오면 몸이 먼저 자라고 반응해서 종필님 아이디어 좋습니다ㅎㅎ

**Comment by jeeyn** [comment]
집중도 있는 논의를 위해 아래와 같이 챕터별로 진행하는 방식은 어떨지 제안해봅니다!
- 기존처럼 PR 늦게 올린 순서대로 진행하되 챕터별 2명씩 진행
  - ex.  ch01 (1, 2번) > ch02 (3, 4번) > ch03 (5, 6번)
- 총 8명이므로 마지막 2명(7, 8번)은 원하는 챕터 마지막에 이어서 발표
  - ex. 7번이 ch01, 8번이 ch03을 발표하고 싶다면
  - ch01 (1, 2, 7번) > ch02 (3, 4번) > ch03 (5, 6, 8번) 순서로 발표
- 때에 따라 챕터 수가 조금씩 달라지므로 챕터별로 진행하되 인원은 상황에 맞게 조정

**Reply by jongfeel** [reply]
챕터별로 정해진 인원이 발표하는 것도 좋은 아이디어 인 것 같습니다.
